packages/core/src/client.ts

```diff
@@ -224,12 +231,17 @@ export class Client<
       const isAuthorized = await connector.isAuthorized()
       if (!isAuthorized) continue
 
-      const data = await connector.connect()
+      let data: ConnectorData
+      try {
+        data = await connector.connect()
+      } catch {
+        // Swallow connect errors
```

Comment:
Was this throwing before in some situations?

Reply:
Yeah, it was throwing unexpectedly when attempting to switch chains... Not sure if we need to throw errors in `autoConnect` if we are trying to eagerly connect?

Reply:
Can we disable chain switching during auto connect? Getting a wallet prompt when the page is loading seems like an anti-pattern.